python/ray/serve/gradio.py

```diff
@@ -0,0 +1,14 @@
+import gradio as gr
+import starlette
+
+from ray.serve.http_util import ASGIHTTPSender
+
+
+class GradioIngress:
```

Comment:
please add a docstring for this class and indicate that it's user-facing. we should also add a `PublicAPI` annotation saying that it's alpha/experimental. See `api.py` for an example